### DISCUSSION TITLE
How to debug source code?

### DISCUSSION THREAD

**Discussion by Kation** [xamarin/Xamarin.Forms#11085] [General]
I'm building Xamarin.Forms myself and want to debug into it.
Create a breakpoint as usually.
When application running to a breakpoint, it was paused but current call stack is not at breakpoint.
It's showing code from current solution' project where first in call stack.
All Xamarin Forms method in **call stack are disabled**.
Both android and ios have the same problem.

I build package by `.create-nuget.bat`.

**It was correct in last few months before I pull source code from git some day.**

### Basic Information

- Version with issue:
- Last known good version:
- IDE: VS2019.6.0
- Platform Target Frameworks: 
  - iOS:  13.5
  - Android: 10
- Nuget Packages: Local build from 4.6.0 branch
- Affected Devices:

### Screenshots

![20200617172632](https://user-images.githubusercontent.com/2033461/84880999-dd85d880-b0bf-11ea-9e36-ccec33e99233.png)
![20200617172728](https://user-images.githubusercontent.com/2033461/84881005-deb70580-b0bf-11ea-86ca-735733a2b7e6.png)

**Comment by hartez** [comment]
@Kation What are the Linker settings for the project you are trying to debug?

**Reply by Kation** [reply]
@hartez All of them can not debug to Forms source code.
